_maps/RandomRuins/IceRuins/icemoon_underground_abandoned_plasma_facility.dmm

```diff
@@ -358,7 +358,7 @@
 /turf/open/lava/plasma/ice_moon,
 /area/icemoon/underground/explored)
 "gd" = (
-/mob/living/simple_animal/hostile/asteroid/wolf,
+/mob/living/basic/mining/wolf,
```

Comment:
I'd actually avoid making them different for ruins if they both exist, mostly given they're static spawns rather than a natural one if that makes sense